### ISSUE TITLE
Should we set the resistance to zero at the end of the course?

### ISSUE THREAD

**Issue by jovial** [fluxoid-org/CyclismoProject#73]
Can result in a sudden change in resistance, as exemplified in #72.

**Comment by dougszumski** [comment]
+1 for leaving it unchanged, then running a warm down routine.

**Comment by jovial** [comment]
Yeah, sounds good. We need to make sure we keep a connection to turbo trainer beyond the end of the course then.
